modules/deleting-a-model-server.adoc

```diff
@@ -4,31 +4,35 @@
 = Deleting a model server
 
 [role='_abstract']
-You can delete model servers that you have assigned to host your data science projects. This enables you to remove model servers that you no longer require. If you delete a project's model server, models hosted by the server are then unavailable for applications to use. 
-ifndef::upstream[]
-Red Hat recommends that you advise the users of these models before deleting the model server. 
-endif::[]
+When you no longer need a model server to host models, you can remove it from your data science project. 
+
+NOTE: When you remove a model server, you also delete the models that are hosted on that model server and they are no longer available to applications.
```

Comment:
Regarding the note here:

"... you also delete the models that are hosted on that model server..." 

I assume that you mean that, any models hosted on the deleted model server are automatically deleted, and are therefore no longer available to applications. 

Also, the end of the note says "...and they are no longer available to applications."

Reply:
I didn't write the original text, but yes, I believe that's the intended meaning.